src/data/modifiers.txt

```diff
@@ -4105,53 +4145,54 @@ Item	anniversary pewter cape	Muscle: +1, Mysticality: +1, Moxie: +1
 Item	bakelite backpack	Monster Level: +10, Candy Drop: +20
+# bakelite backpack: Makes Accordion Bash knock loose some Meat
 Item	balsam barrel	Muscle Percent: +50, Mysticality Percent: +50, Moxie Percent: +50, Item Drop: +15, HP Regen Min: 10, HP Regen Max: 20, MP Regen Min: 10, MP Regen Max: 20
 Item	barskin cloak	Muscle: +7
 Item	black cloak	Spell Damage: +10
 Item	bony back shell	Spooky Resistance: +2, Stench Resistance: +2, Hot Resistance: +2, Cold Resistance: +2, Sleaze Resistance: +2
+Item	Buddy Bjorn	Softcore Only
 # Buddy Bjorn: Put a Familiar In It!
 # Buddy Bjorn: That Familiar gains 1 Experience per Fight
-Item	Buddy Bjorn	Softcore Only
 Item	burning cape	Hot Resistance: +3, Maximum HP: +25, Maximum MP: +25, Adventures: +5, Lasts Until Rollover
-# Camp Scout backpack: Helps You Be Prepared
 Item	Camp Scout backpack	Item Drop: +15, Softcore Only, Drops Items
+# Camp Scout backpack: Helps You Be Prepared
 Item	cape	Initiative: +100
-Item	Catherine Wheel	Muscle Percent: +20, Mysticality Percent: +20, Moxie Percent: +20, Experience: +3, Lasts Until Rollover
 Item	carpe	Mysticality Percent: +50, MP Regen Min: 5, MP Regen Max: 10, Meat Drop: +50, Combat Rate: +10, Lasts Until Rollover
-# chalk chlamys: Weakens enemies who hit you
+Item	Catherine Wheel	Muscle Percent: +20, Mysticality Percent: +20, Moxie Percent: +20, Experience: +3, Lasts Until Rollover
 Item	chalk chlamys	Adventures: +2, Combat Rate: -5
+# chalk chlamys: Weakens enemies who hit you
 Item	Cloak of Dire Shadows	Maximum MP: +40, Meat Drop: +10, Muscle Percent: +5, Mysticality Percent: +5, Moxie Percent: +5
 Item	cod cape	Stench Damage: +25, Combat Rate: +5
 Item	cursed blanket	Hot Resistance: +3, Cold Resistance: +3, Spooky Resistance: +3, Stench Resistance: +3, Sleaze Resistance: +3, Familiar Weight: -20
+Item	cursed goblin cape	Combat Rate: -15, Cold Resistance: -6, Spooky Resistance: -6
 # cursed goblin cape: Cold Vulnerability (-6)
 # cursed goblin cape: Spooky Vulnerability (-6)
-Item	cursed goblin cape	Combat Rate: -15, Cold Resistance: -6, Spooky Resistance: -6
 Item	dice-shaped backpack	Random Monster Modifiers: +1
 Item	Drapes-You-Regally	Meat Drop: +20, Booze Drop: +20, Hat Drop: +20, Class: "Disco Bandit"
-# Drip harness: Allows you to enter The Drip
 Item	Drip harness	Muscle Limit: 100, Mysticality Limit: 100, Moxie Limit: 100
+# Drip harness: Allows you to enter The Drip
 Item	Drunkula's cape	Adventures: +3, Candy Drop: +20, Accessory Drop: +20, Class: "Sauceror"
 Item	Duke Vampire's regal cloak	Moxie: +15, Moxie Percent: +25, Initiative: +50
 Item	elf army poncho	Spooky Resistance: +2, Stench Resistance: +2, Hot Resistance: +2, Cold Resistance: +2, Sleaze Resistance: +2, Lasts Until Rollover
-# fiberglass frock: Makes it so dealing Cold damage causes ongoing additional Cold damage
+Item	Elf Guard SCUBA tank	Muscle Percent: -25, Mysticality Percent: -25, Moxie Percent: -25, Maximum HP: -100, Maximum MP: -100, Adventure Underwater, Adventure Underwater
```

Comment:
while what Mafia returns, Adventure Underwater should not be duplicated.

It's duplicated because the description looks like
```
Lets you breathe underwater
Lets you Breathe Underwater
```
which is presumably a KoL bug